templates/openstack/managed.html

```diff
@@ -1,725 +1,618 @@
+          <div class="col-6">
+    
+            <h3 class="p-stepped-list__title">
+              Transfer
+            </h3>
+            <p>On request, we transfer control over your cloud back to your organisation. We
+              train your personnel, make sure you are able to operate the cloud yourself moving forward and perform a formal
+              handover process. We continue providing commercial support to make sure your cloud receives security updates
+              and you receive help if needed.</p>
+          </div>
+          <div class="col-6 u-align--center u-vertically-center u-hide--small">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/2131b805-We+transfer+control.svg",
+                alt="",
+                height="134",
+                width="134",
+                hi_def=True,
+                loading="lazy",
+              ) | safe
+            }}
+          </div>
+        </div>
+      </li>
+    </ol>
   </div>
+</section>
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Managed OpenStack pricing</h2>
+  </div>
+  <div class="row">
+    <div class="col-6 p-card">
+      <h3 class="p-heading--4">$15 per server per day</h3>
+      <hr class="u-sv1" />
+      <p>What's included:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Logging, monitoring and alerting</li>
+        <li class="p-list__item is-ticked">24x7 operations with fully staffed NOC</li>
+        <li class="p-list__item is-ticked">Proactive cloud management</li>
+        <li class="p-list__item is-ticked">Incident and problem resolution</li>
+        <li class="p-list__item is-ticked">Software updates and OpenStack upgrades</li>
+        <li class="p-list__item is-ticked">SLA and regular independent audits</li>
+        <li class="p-list__item is-ticked">Most cost-effective approach up to 300 nodes</li>
+        <li class="p-list__item is-ticked">Optional Managed Kubernetes service on top</li>
+        <li class="p-list__item is-ticked">EU and US regulatory compliance options</li>
+      </ul>
+    </div>
+  </div>
+</section>
 
+<section class="p-strip--light">
+  <div class="row">
+    <h2>Modernise your infrastructure with Managed OpenStack</h2>
+    <div class="col-8">
+      <p>Canonical's Managed OpenStack brings your infrastructure costs down, while not putting an overhead on your
+        internal operations team. Talk to one of our cloud experts to get cost estimates for Managed OpenStack and choose
+        the package tailored to your needs.</p>
+      <p><a href="/openstack/contact-us" class="p-button--positive">Talk to an expert</a></p>
+    </div>
+    <div class="col-4 u-hide--small u-align--center">
+      {{
+      image(
+      url="https://assets.ubuntu.com/v1/24a03775-Contact+us.svg",
+      alt="",
+      height="171",
+      width="240",
+      hi_def=True,
+      loading="lazy",
+      ) | safe
+      }}
+    </div>
+  </div>
```

Comment:
@minkyngkm is right, but you'd need a couple extra classes on the `col-4` to vertically center the image. See the section titled "OpenStack ensures maximum multi-cloud cost optimisation" on https://ubuntu.com/openstack/what-is-openstack for another example.

```suggestion
  <div class="row">
    <div class="col-8">
      <h2>Modernise your infrastructure with Managed OpenStack</h2>
      <p>Canonical's Managed OpenStack brings your infrastructure costs down, while not putting an overhead on your
        internal operations team. Talk to one of our cloud experts to get cost estimates for Managed OpenStack and choose
        the package tailored to your needs.</p>
      <p><a href="/openstack/contact-us" class="p-button--positive">Talk to an expert</a></p>
    </div>
    <div class="col-4 u-align--center u-vertically-center u-hide--small">
      {{
      image(
      url="https://assets.ubuntu.com/v1/24a03775-Contact+us.svg",
      alt="",
      height="171",
      width="240",
      hi_def=True,
      loading="lazy",
      ) | safe
      }}
    </div>
  </div>
```